src/worker.js

```diff
@@ -10,41 +10,26 @@ import { isConfigAtHomeRoot } from './is-config-at-home-root'
 
 process.title = 'linter-eslint helper'
 
-const ignoredMessages = [
-  // V1
-  'File ignored because of your .eslintignore file. Use --no-ignore to override.',
-  // V2
-  'File ignored because of a matching ignore pattern. Use --no-ignore to override.',
-  // V2.11.1
-  'File ignored because of a matching ignore pattern. Use "--no-ignore" to override.',
-  // supress warning that the current file is ignored by eslint by default
-  'File ignored by default.  Use a negated ignore pattern (like "--ignore-pattern \'!<relative'
-    + '/path/to/filename>\'") to override.',
-  'File ignored by default. Use "--ignore-pattern \'!node_modules/*\'" to override.',
-  'File ignored by default. Use "--ignore-pattern \'!bower_components/*\'" to override.',
-]
+function lintJob({ cliEngineOptions, contents, eslint, filePath }) {
+  const cliEngine = new eslint.CLIEngine(cliEngineOptions)
```

Comment:
I wonder, are we going to pay a performance penalty here by creating a new `cliEngine` each time we perform a linting job?  If there's much overhead here, maybe it makes sense to memoize it, and only create a new `cliEngine` when the options change.  What do you think?